### ISSUE TITLE
Refactor `sections` property of `KLSampleSheet` to reduce confusion with `_sections` property of parent class

### ISSUE THREAD

**Issue by AmandaBirmingham** [biocore/kl-metapool#233]
`KLSampleSheet` includes the property:

```
    sections = (_HEADER_KEY, _READS_KEY, _SETTINGS_KEY, _DATA_KEY,
                _BIOINFORMATICS_KEY, _CONTACT_KEY)
```

I would love to know if this is being used outside this module or not and if not would love to rename it, as the underlying Illumina-code `SampleSheet` object has a `_sections` attribute, which is a little tooclose for comfort (also, the sheet's `add_section()` method adds to *that* attribute, but one might be forgiven for guessing it would add to this one .... )

**Comment by charles-cowart** [comment]
I don't believe that it is, at least not in mg-scripts or the SPP plugin. Even if it was it's not a hard fix. Regarding our earlier discussion about applying an Illumina-based file-spec to other types of processes, I've been wondering if removing sample_sheet as a dependency entirely and storing the values for the [Data] section in a Pandas DataFrame might be beneficial.

**Comment by AmandaBirmingham** [comment]
> I've been wondering if removing sample_sheet as a dependency entirely and storing the values for the [Data] section in a Pandas DataFrame might be beneficial.

Yeah, honestly I'd love that ... I think it would make our lives easier to separate out *our* use of sample sheets (as a vehicle for transmitting info for our own data processing purposes, which will be needed even for completely different sequencing technologies) from Illumina's use of sample sheets.  I mean, obviously at some point an "us" sample sheet would have to be convertible to/readable as an llumina sample sheet, but I think we could profitably lose the inheritance relationship here :)
